test.js

```diff
@@ -0,0 +1,63 @@
+const { readdirSync } = require('fs');
+const { join } = require('path');
+const exec = require('async-execute');
+
+/**
+ * fixtures directory
+ * @type {string}
+ */
+const fixtures = join(__dirname, 'fixtures');
+
+/**
+ * eslint binary
+ * @type {string}
+ */
+const eslint = join(__dirname, 'node_modules/.bin/eslint');
+
+/**
+ * this test case config file
+ * @type {string}
+ */
+const config = join(__dirname, '.eslintrc');
+
+/**
+ * Output of stream output
+ * @type {object}
+ */
+const result = {
+    status: undefined
```

Comment:
No need to store this outside. You can have your lint result resolve or reject with the message.

Reply:
fine